admin/bower.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "OSF_admin",
+  "dependencies": {
+    "jquery": "^1.11.2",
```

Comment:
Is it necessary to copy all the bower dependencies from OSF core? Could you instead use the assets that are already installed? Y